data/wanderer/wanderers middle.txt

```diff
@@ -3413,17 +3413,17 @@ mission "Wanderers: Sestor: Final: Patched"
 		has "Wanderers: Sestor: Final: offered"
 	waypoint "Kor Fel'tar"
 	on enter "Kor Fel'tar"
-		dialog `There are several Kor Mereti and Kor Sestor drones in this system, and they don't look friendly. You should report to the Wanderers that the exiles have begun manufacturing drones of their own.`
+		dialog `There are several Kor Mereti and Kor Sestor drones in this system, and they don't look friendly. You should report to the Wanderers that the Exiles have begun manufacturing drones of their own.`
 	on offer
 		event "wanderers: exiles hostile" 4
 		conversation
 			`When you meet with the Wanderer leaders, Sobari Tele'ek tells you, "We have received word from our [scouts, observers] that the Sestor drones are [dwindling, reducing] in numbers and will soon be no more. We owe you our [gratitude, thanks]. But, the Mereti tell us a more worrisome [tale, story]."`
-			`	Meto Pa'aret says, "The Mereti [contacted, communicated with] me and said that one of their stations was [raided, infiltrated] by the Korath exiles, and some of their [machinery, equipment] was stolen. The Mereti did not [fight, attack] the exiles because they thought they were [friends, allies]. Captain <last>, can you bring Rek to visit the [exiles, outcasts] and learn what their plans are?"`
+			`	Meto Pa'aret says, "The Mereti [contacted, communicated with] me and said that one of their stations was [raided, infiltrated] by the Korath Exiles, and some of their [machinery, equipment] was stolen. The Mereti did not [fight, attack] the Exiles because they thought they were [friends, allies]. Captain <last>, can you bring Rek to visit the [Exiles, outcasts] and learn what their plans are?"`
```

Comment:
I think this one was fine as-is, given the nature of the translator. If it was clear to the translator that they were talking about the Korath Exiles, it wouldn't be appearing this way.

Reply:
Sure I'll change that